besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1089,6 +1089,13 @@ void setBannedNodeIds(final List<String> values) {
           "Specifies the static node file containing the static nodes for this node to connect to")
   private final Path staticNodesFile = null;
 
+  @SuppressWarnings({"FieldCanBeFinal", "FieldMayBeFinal"}) // PicoCLI requires non-final Strings.
+  @CommandLine.Option(
+          names = { "--dns-discovery-url" },
```

Comment:
Hmm... this feels like discovery should come first.  perhaps `--discovery-dns-url`.  @NicolasMassart any opinions on this or ideas on who it should be run by?